internal/api/internal.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
```

Comment:
I think `statestore.proto` might be a more appropriate name for this file. It matches the style of using the module name in the proto file name that `synchronizer.proto` has started.

Reply:
messages.proto would also be appropriate, as we put all message definitions that aren't request/response types in a similarly named file in the public api.

Reply:
Oh that file is actually being added by https://github.com/googleforgames/open-match/pull/1273

Reply:
My initial take on how this file should be named was not right. Laramere educated me on how these files are named. The naming pattern is not based on the module name. Instead, services and their req/resp usually go into their own proto file, and objects that aren't coupled to a service's API usually go in a messages.proto file. 
+1 for messages.proto

Reply:
I will copy a following thread back to the author @akremsa